tests/unit/__init__.py

```diff
@@ -70,7 +75,13 @@ class AppArgs:
 # NVDAObjects need appModuleHandler to be initialized.
 import appModuleHandler
 appModuleHandler.initialize()
-# Anything which notifies of cursor updates requires braille to be initialized.
+# Vision needs a wx app to be available.

```

Comment:
From a unit testing perspective it would be better if we can avoid this. Which part of the vision framework relies on a wx app being available? Can we mock this part instead?